inc/renderer.php

```diff
@@ -261,47 +261,49 @@ function render( $post_id = false, $enqueue_css = true, $panels_data = false ) {
-	 * @param array $style_args The style wrapper args. Used as an argument for siteorigin_panels_{$name}_style_attributes
+	 * @param array $style The style wrapper args. Used as an argument for siteorigin_panels_{$name}_style_attributes
 	 * @param string|bool $for An identifier of what this style wrapper is for
 	 *
 	 * @return bool Is there a style wrapper
 	 */
-	private function start_style_wrapper( $name, $style_args = array(), $for = false ) {
+	private function start_style_wrapper( $name, $style = array(), $for = false ) {
+		$attributes = array();
 
-		$style_wrapper = '';
-
-		if ( empty( $style_attributes['class'] ) ) {
-			$style_attributes['class'] = array();
+		if ( empty( $attributes['class'] ) ) {
+			$attributes['class'] = array();
 		}
-		if ( empty( $style_attributes['style'] ) ) {
-			$style_attributes['style'] = '';
+		if ( empty( $attributes['style'] ) ) {
+			$attributes['style'] = '';
 		}
 
-		$style_attributes = apply_filters( 'siteorigin_panels_' . $name . '_style_attributes', $style_attributes, $style_args );
+		$attributes = apply_filters( 'siteorigin_panels_' . $name . '_style_attributes', $attributes, $style );
 
-		if ( empty( $style_attributes['class'] ) ) {
-			unset( $style_attributes['class'] );
+		if ( empty( $attributes['class'] ) ) {
+			unset( $attributes['class'] );
 		}
-		if ( empty( $style_attributes['style'] ) ) {
-			unset( $style_attributes['style'] );
+		if ( empty( $attributes['style'] ) ) {
+			unset( $attributes['style'] );
 		}
 
-		if ( ! empty( $style_attributes ) ) {
-			if ( empty( $style_attributes['class'] ) ) {
-				$style_attributes['class'] = array();
+		$style_wrapper = '';
+		if ( ! empty( $attributes ) ) {
+			if ( empty( $attributes['class'] ) ) {
+				$attributes['class'] = array();
 			}
-			$style_attributes['class'][] = 'panel-' . $name . '-style';
+			$attributes['class'][] = 'panel-' . $name . '-style';
 			if ( ! empty( $for ) ) {
-				$style_attributes['class'][] = 'panel-' . $name . '-style-for-' . sanitize_html_class( $for );
+				$attributes['class'][] = 'panel-' . $name . '-style-for-' . sanitize_html_class( $for );
 			}
-			$style_attributes['class'] = array_unique( $style_attributes['class'] );
+			$attributes['class'] = array_unique( $attributes['class'] );
 
 			// Filter and sanitize the classes
-			$style_attributes['class'] = apply_filters( 'siteorigin_panels_' . $name . '_style_classes', $style_attributes['class'], $style_attributes, $style_args );
-			$style_attributes['class'] = array_map( 'sanitize_html_class', $style_attributes['class'] );
+			$attributes['class'] = apply_filters( 'siteorigin_panels_' . $name . '_style_classes', $attributes['class'], $attributes, $style );
+			$attributes['class'] = array_map( 'sanitize_html_class', $attributes['class'] );
 
 			$style_wrapper = '<div ';
-			foreach ( $style_attributes as $name => $value ) {
+			foreach ( $attributes as $name => $value ) {
+				if( substr( $name, 0, 1 ) === '_' ) continue;
```

Comment:
Is there a special type of attribute that starts with an underscore? Something I'm usually quite bad with is leaving clarifying comments  for special cases like this, so will try to point these out when I see them and fix them in my own code too.